src/ServiceProvider.php

```diff
@@ -197,7 +197,7 @@ protected function getAgentConfig(): array
         // Filter out null config options so that the Config class can look for environment variables
         return array_filter(array_merge(
             [
-                'defaultServiceName' => 'Laravel',
+                'defaultServiceName' => config('elastic-apm-laravel.app.appName', 'Laravel'),
```

Comment:
`Laravel` is already the default value in that config.

Reply:
Users could remove the default though in their published config. Not harm on having yet another default I guess.